Sources/SwiftDoc/Symbol.swift

```diff
@@ -225,7 +228,7 @@ extension Symbol: Codable {
     private enum CodingKeys: String, CodingKey {
         case declaration
         case documentation
-        case sourceLocation
+        case sourceRange
```

Comment:
🤔 This is also a topic for the "Do we really offer SwiftDoc as a standalone library?" discussion. If yes, we would need to document changes of the serialization of public objects.

Reply:
Yeah, that's a fair point. (Although, to be honest, I don't recall why `Symbol` is `Codable` in the first place...)

I added changelog entries with 11b21ed. Anything else to be done here?

Reply:
Early preparations for caching or for getting symbols from different sources than source files?

For the long-running discussion outside the scope of this pull request: I think removing `Codable` conformance is a reasonable option. It's not needed for swift-doc and adds maintenance effort. Users of the library could implement it themselves.

For this pull request: Looks good to me, but there's now a conflict because of the changelog changes.

Reply:
> Early preparations for caching or for getting symbols from different sources than source files?

Yeah, that was probably it. Let's leave things as they are for now. We can revisit all of that when we get to vending serialized representations (such as JSON or RDF).